src/sbuf.c

```diff
@@ -882,39 +884,52 @@ static void setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 		err = tls_config_parse_protocols(&protos, protocols);
 		if (err) {
 			log_error("invalid %s_protocols: %s", pfx, protocols);
+			return false;
```

Comment:
I have committed these error handling changes in this function separately (4c474cabd24c0bb5d85d3e8088d82a5ad0fa3da6), since they fit with some other work to make configuration error handling more robust.

Reply:
Sounds good, rebased these changes on top of the current master branch.